validation/src/test/scala/hmda/validation/rules/ts/validity/V120Spec.scala

```diff
@@ -0,0 +1,38 @@
+package hmda.validation.rules.lar.validity
+
+import hmda.model.fi.ts.TransmittalSheet
+import hmda.validation.rules.EditCheck
+import hmda.validation.rules.ts.TsEditCheckSpec
+import hmda.validation.rules.ts.validity.V120
+import org.scalacheck.Gen
+
+class V120Spec extends TsEditCheckSpec {
+
+  property("passes with properly formed number") {
+    forAll(tsGen) { ts =>
+      whenever(ts.contact.phone.length == 12) {
```

Comment:
It looks like (with the new generator function), phone numbers will never != 12.  Have you tried taking this `whenever` statement out and running the tests?